tests/test_tiproviders.py

```diff
@@ -404,11 +404,11 @@ def test_tor_exit_nodes(self):
             else:
                 neg_results.append(lu_result)
 
-        self.assertEqual(len(pos_results), 4)
```

Comment:
I think I fixed this in iputils - since this list is going to be changing I'm taking the known Tor nodes as a sample from the current list and testing against that - so unless one of the neg-results set becomes a Tor node the test should continue to work. We can deal with this when we merge though.